cluster-autoscaler/core/scaleup/orchestrator/async_initializer.go

```diff
@@ -144,7 +147,17 @@ func (s *AsyncNodeGroupInitializer) InitializeNodeGroup(result nodegroups.AsyncN
 			failedNodeGroupIds = append(failedNodeGroupIds, failedNodeGroup.Id())
 		}
 		klog.Errorf("Async scale-up for asynchronously created node group failed: %v (node groups: %v)", err, failedNodeGroupIds)
+		s.emitScaleUpStatus(&status.ScaleUpStatus{
```

Comment:
is there any existing UT surface area where we could mock this scenario and detect the status update?

Reply:
Just added the missing UT, thanks.